ishimweClaude12/action-in-action
# fix swagger documantation

# Feature name:
fix-swagger

## Description:
The main purpose of this feature is to update the structure of the  swagger document used to describe how end points works 

## Testing:
- configure your env file based on .env.example file
- you should run npx tsc to generate dist file  if you don't already have it
- run npm run dev to start you project 
- Access  http://localhost:4000/api-docs to see the document

## Additional information:
- Everything is not working well as it must be, you should check everything to make it smooth based on your controller functionality.
- Implement the other end points
